tools/c7n_mailer/c7n_mailer/msg-templates/slack_good.j2

```diff
@@ -0,0 +1,37 @@
+{
+   "attachments":[
+      {
+         "fallback":"Cloud Custodian Policy Violation",
+         "title":"Custodian",
+         "color":"good",
```

Comment:
why not just make this field a variable instead of copy and pasting the template multiple times

Reply:
Path of least resistance :).

I'd be happy to make it a variable. I'll update this PR in the near future.